projects/control-service/projects/job-builder-secure/version.txt

```diff
@@ -1 +1 @@
-1.0.0
+1.1.0
```

Comment:
nit: do we still use the version.txt mechanism or is it redundant?

Reply:
we use it for job-builder